### PR TITLE
fix(web): search function on Asset or Request

### PR DESCRIPTION
**Pull request by caichi-t**
# Overview
This PR includes resetting the page after searching on Asset (also Modal) or Request and resetting the input value after closing LinkAssetModal.

### CODE REVIEW

File: web/src/components/molecules/Asset/AssetListTable/index.tsx [hunk 1]
```diff
@@ -82,38 +80,36 @@ const AssetListTable: React.FC<AssetListTableProps> = ({
...
       },
+      align: "center",
     },
     {
       title: t("File"),
       dataIndex: "fileName",
       key: "NAME",
       sorter: true,
-      defaultSortOrder:
-        sort?.type === "NAME" ? (sort.direction === "ASC" ? "ascend" : "descend") : null,
```
Comment (nourbalaha) [inline web/src/components/molecules/Asset/AssetListTable/index.tsx:107]: Why did you remove the default sort order?

Comment (caichi-t) [reply web/src/components/molecules/Asset/AssetListTable/index.tsx:107]: I didn't think it needed to be displayed as well as the Content table, should I change it back?

Comment (nourbalaha) [reply web/src/components/molecules/Asset/AssetListTable/index.tsx:107]: Okay if it is not displayed then it is okay to remove it.

**Review by nourbalaha — APPROVED** [review]
